index.html

```diff
@@ -47,25 +47,287 @@ <h1 class="navbar-dropdown-item">
     </nav>
     <section class="top-hero-section">
         <div class="top-hero-container">
-                <div>
-                    <h1 class="hero-heading">Build better processes</h1>
+            <div>
+                <h1 class="hero-heading">Build better processes</h1>
+            </div>
+            <div>
+                <p class="hero-paragraph">Your customers won’t settle and neither should you. With Pluralsight,
+                    technology teams know more
+                    and
+                    work
+                    better together. Develop the skills you need for your top priorities. Improve workflows with
+                    actionable
+                    data. Build better—every time.
+                </p>
+            </div>
+            <div class="hero-button-container">
+                <div class="hero-button-size">
+                    <button class="view-plans-button">VIEW PLANS</button>
                 </div>
                 <div>
-                    <p class="hero-paragraph">Your customers won’t settle and neither should you. With Pluralsight,
-                        technology teams know more
-                        and
-                        work
-                        better together. Develop the skills you need for your top priorities. Improve workflows with
-                        actionable
-                        data. Build better—every time.
+                    <button class="try-for-free-button">TRY FOR FREE</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- trending technologies-->
+    <section class="trending-technologies-container-wrapper">
+        <div class="container-wrapper">
+            <div class="trending-technologies-main-container">
+                <div class="trending-technologies-container">
+                    <h6 class="title-pink">TRENDING TECHNOLOGIES</h6>
+                    <h3 class="title-sub">Popular topics
+                        to learn now</h3>
+                    <p>
+                        Not sure what technologies to focus on? The Technology Index ranks 850+ technologies by their
+                        relative
+                        popularity. See what’s topping the charts and get recommended courses to skill up.
                     </p>
+                    <div class="arrow-path-container">
+                        <div class="arrow-path-text">
+                            <p>View the index</p>
+                        </div>
+                        <div class="right-arrow">
```

Comment:
it is showing here .is it not showing on your browser? The chevron-right.svg

Reply:
I think it is not showing because your path is not pointing correctly. Make sure to use "./" prefix before specifing the path because this makes it a relative path.

Reply:
./images/chevron-right.svg .I have used this path.please recheck